lib/Models/Catalog/SdmxJson/SdmxJsonCatalogItem.ts

```diff
@@ -112,12 +114,11 @@ export default class SdmxJsonCatalogItem
    */
   @computed
   get baseUrl(): string | undefined {
-    return super.url;
+    return this.traits["url"].getValue(this);
```

Comment:
`super.url` no longer returns the non-overridden trait value; it invokes `urlOverride` and returns the exact same thing as `this.url`. So we need to directly to the trait to get its value. IMO this should be refactored so that the `url` trait can be the normal URL, and a separate property provides the derived data request URL. But I decided not to get involved.